src/runtime_src/core/common/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ file(GLOB XRT_CORE_COMMON_LIB_FILES
 if (DEFINED XRT_AIE_BUILD)
   file(GLOB XRT_CORE_COMMON_AIE_LIB_FILES
     "api/aie/*.cpp"
-    )
+  )
```

Comment:
Aren't api/aie/*.cpp supposed to be built?

Reply:
Its just a space change, not removed it Soren

Reply:
we have added the define XRT_AIE_BUILD in src/CMakeLists.txt when XRT_NATIVE_BUILD is set to yes. So it ideally builds even for the x86

Reply:
No it is not.   You didn't set XRT_AIE_BUILD for CMake, you set it for g++.

Reply:
Thanks for correcting me Soren, I did not realize its only for GCC, Let me correct as you said and verify the changes before submit.

Reply:
No problem.   Yes, please validate a few examples before pushing the changes again.  You are close now :-)